ocs_sample_library_preview/DataView/ResolvedDataItems.py

```diff
@@ -0,0 +1,58 @@
+import json
```

Comment:
whats the reason for the change to Resolved?  The base DataItem also works for ineligible dataitems

Reply:
Mainly just to follow the naming pattern we use internally. The internal generic class is `ResolvedItems<T>`. Calling it `DataItems` didn't seem right to me, but I also didn't want to convert this to a fully generic type, so I went with `ResolvedDataItems`.